docs/discovery.md

```diff
@@ -97,7 +97,9 @@ swarm join \
     consul://<consul_addr>/<optional path prefix>
 ```
 
-This works the same way for the Swarm `manage` and `list` commands.
+This works the same way for the Swarm `manage` and `list` commands, but
+remember, if you are running the swarm image rather than the binary the paths
```

Comment:
`if you are running the swarm image` -> `if you are running swarm inside a container`